blocks/ai-text/src/app/generate-text/model-selector.tsx

```diff
@@ -0,0 +1,321 @@
+import { MenuItem, Select } from "@hashintel/design-system";
+import {
+  Box,
+  inputBaseClasses,
+  outlinedInputClasses,
+  Stack,
+  Typography,
+} from "@mui/material";
+import { ReactNode, useMemo, useState } from "react";
+
+import { AbstractAiIcon } from "../../icons/abstract-ai";
+
+export const DEFAULT_MODEL_ID = "gpt-3.5-turbo";
+
+enum ModelGroupName {
+  OpenAi = "OpenAi",
+}
+
+interface ModelGroup {
+  name: string;
+  icon: ReactNode;
+  models: Model[];
+}
+
+interface Model {
+  id: string;
+  name: string;
+  description: string;
+}
+
+const MODELS_BY_GROUP: { [key in ModelGroupName]: ModelGroup } = {
+  [ModelGroupName.OpenAi]: {
+    name: ModelGroupName.OpenAi,
+    icon: <AbstractAiIcon sx={{ fontSize: "inherit" }} />,
+    models: [
+      {
+        id: "gpt-3.5-turbo",
+        name: "GPT-3.5 Turbo",
+        description:
+          "The best current language model for many use cases. Newer, faster and 10x cheaper than Davinci.",
```

Comment:
```suggestion
          "The best current language model for many use cases; newer, faster and 10x cheaper than Davinci",
```

Reply:
The `.` at the end was the other change here that got missed... however I have managed to shorten this a bit more. See https://github.com/hashintel/hash/pull/2176/files#r1129563430